BurstBotNET/Commands/NinetyNine/Game.cs

```diff
@@ -301,63 +323,115 @@ ILogger logger
-                            playerId, sendResult.Error.Message, sendResult.Inner);
+                            playerId, result.Error.Message, result.Inner);
+
                     continue;
                 }
-                var result = await channelApi
+
+                var confirmComponent = BuildConfirmButton(localization);
+
+                embed = embed with
+                {
+                    Description = localization.GameOver +
+                                  $"\n\n{localization.Cards}" +
+                                  $"\n\n{string.Join('\n', nextPlayer.Cards)}" +
+                                  $"\n\n{localization.CurrentTotal.Replace("{total}", deserializedIncomingData.CurrentTotal.ToString())}",
+                    Image = new EmbedImage(Constants.AttachmentUri)
+                };
+
+                var sendConfirmButtonResult = await channelApi
                     .CreateMessageAsync(playerState.TextChannel.ID,
                         embeds: new[] { embed },
-                        attachments: attachment,
-                        components: component);
-                if (!result.IsSuccess)
+                        components: confirmComponent);
+
+                if (!sendConfirmButtonResult.IsSuccess)
                     logger.LogError("Failed to send drawing message to player {PlayerId}: {Reason}, inner: {Inner}",
-                        playerId, result.Error.Message, result.Inner);
+                        playerId, sendConfirmButtonResult.Error.Message, sendConfirmButtonResult.Inner);
             }
             else
             {
-                var sendResult = await channelApi
-                    .CreateMessageAsync(playerState.TextChannel.ID,
-                        embeds: new[] { embed });
-                if (!sendResult.IsSuccess)
-                    logger.LogError("Failed to send drawing message to player {PlayerId}: {Reason}, inner: {Inner}",
-                        playerId, sendResult.Error.Message, sendResult.Inner);
+                if (renderedHiddenImage != null)
+                {
+                    var randomFileName = Utilities.GenerateRandomString() + ".jpg";
+                    var randomAttachmentUri = $"attachment://{randomFileName}";
+
+                    await using var imageCopy = new MemoryStream((int)renderedHiddenImage.Length);
+                    await renderedHiddenImage.CopyToAsync(imageCopy);
+                    imageCopy.Seek(0, SeekOrigin.Begin);
+                    renderedHiddenImage.Seek(0, SeekOrigin.Begin);
+
+                    embed = embed with
+                    {
+                        Image = new EmbedImage(randomAttachmentUri)
+                    };
+
+                    var attachment = new OneOf<FileData, IPartialAttachment>[]
+                    {
+                        new FileData(randomFileName, imageCopy)
+                    };
+
```

Comment:
Process can be a helper function

Reply:
This was already mentioned here: https://github.com/AllBurst/BurstBotNET/pull/5#discussion_r788107630